base/util.jl

```diff
@@ -134,8 +112,13 @@ function padded_nonzero_print(value,str)
 end
 
 function time_print(elapsedtime, bytes, gctime, allocs)
-    mt, pptime, fraction = prettyprint_nanoseconds(elapsedtime)
-    (fraction != 0) ? @printf("%4d.%03d %s", pptime, fraction, _sec_units[mt]) : @printf("%8d %s",  pptime, _sec_units[mt])
+    seconds = elapsedtime/1e9
+    if 1 <= round(seconds,3)
+        @printf("%.3f seconds", seconds)
+    else
+        power = -floor(log10(seconds))
+        @printf("%.3f/10^%d seconds", seconds*10^power, power)
```

Comment:
I usually see negative powers rather than division to represent small numbers, but perhaps my exposure to scientific notation is too limited.